### PR TITLE
adding helper script

### PR DESCRIPTION
**Pull request by apaul7**
this adds a helper script to create the input file that contains the paths to the vcfs and uncompress them if needed

### CODE REVIEW

File: survivor_merge_helper.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o pipefail
+set -o errexit
+
```
Comment (tmooney) [inline survivor_merge_helper.sh:6]: I'd add `set -o nounset` just as an extra precaution against one of the 10 parameters not being passed.
```suggestion
set -o nounset

```

File: survivor_merge_helper.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# uncompress files as needed
+for i in $(echo ${3} | tr "," "\n"); do
+    if [[ $i == *.gz ]]; then
+        gunzip "$i"
+        i=$(echo "$i" | sed 's/\(.*\)\.gz/\1/')
+    fi
+    echo "$i" >> vcfs.list
+done
+
+echo "/usr/bin/SURVIVOR merge vcfs.list ${4} ${5} ${6} ${7} ${8} ${9} ${10}"
```
Comment (tmooney) [inline survivor_merge_helper.sh]: This last line is missing its newline character!
(The github suggester doesn't seem to work right for suggesting that change.)

**Review by tmooney — COMMENTED** [review]
I think this generally looks fine, but I'm curious why `$3` is the lowest-numbered parameter that gets used in the script.

File: survivor_merge_helper.sh (hunk 1 above)
Comment (apaul7) [reply survivor_merge_helper.sh:6]: added!

**Comment by apaul7** [comment]
I originally had `/usr/bin/SURVIVOR` and `merge` as parameters 1 and 2 in the cwl. but they are not needed because the script sets them

**Review by tmooney — APPROVED** [review]
+1.

How is the file-of-files getting generated?  Depending how this gets plugged into the CWL, that could be something else the helper does.

**Comment by apaul7** [comment]
originally I was generating the files of file paths as part of the cwl in the initial working directory. I've switched the script to handle string of comma separated paths to files. so the cwl will just pass that in instead of a file of paths.

I'm going to merge this so I can test the whole cwl workflow on the cluster

**Pull request merged**